src/libmeasurement_kit/nettests/captive_portal.cpp

```diff
@@ -0,0 +1,25 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+
+#include "private/nettests/runnable.hpp"
+
+#include <measurement_kit/nettests.hpp>
+#include <measurement_kit/ooni.hpp>
+
+namespace mk {
+namespace nettests {
+
+CaptivePortalTest::CaptivePortalTest() : BaseTest() {
+    runnable.reset(new CaptivePortalRunnable);
+    runnable->test_name = "captive_portal";
+    runnable->test_version = "0.0.1";
```

Comment:
Given that we're at format `0.4` in the spec, shouldn't this be `"0.4.0"`, @joelanders?